packages/react-icon-provider/src/createOverride.ts

```diff
@@ -0,0 +1,12 @@
+import { IIconSubset } from '@fluentui/style-utilities';
+
+/**
+ * Creates a custom icon override.
+ * @param icons - Icon map with .
+ */
+export function createIconOverride(iconSet: { [key: string]: string | JSX.Element }): IIconSubset {
```

Comment:
I'm not sure this helper is necessary. Unlike with themes, it's easy for the user to build a correctly-shaped icon subset object themselves.

Reply:
That makes sense. I mainly made it to keep to what appears to be the paradigm for making `Provider` components like `ThemeProvider` and `WindowProvider`